proto/buf/alpha/registry/v1alpha1/labels.proto

```diff
@@ -40,8 +40,12 @@ message LabelValue {
 }
 
 service LabelService {
-  rpc CreateLabel(CreateLabelRequest) returns (CreateLabelResponse);
-  rpc MoveLabel(MoveLabelRequest) returns (MoveLabelResponse);
+  rpc CreateLabel(CreateLabelRequest) returns (CreateLabelResponse) {
+    option idempotency_level = IDEMPOTENT;
+  }
+  rpc MoveLabel(MoveLabelRequest) returns (MoveLabelResponse) {
+    option idempotency_level = IDEMPOTENT;
```

Comment:
Just making sure this is right @doriable

For quick reference: https://developer.mozilla.org/en-US/docs/Glossary/Idempotent

Reply:
Yes, I believe this is correct: you're always required to set the commit the label is moving to, so calling the move to the same destination multiple times would be considered idempotent.

Reply:
I didn't realize that 'from' was optional, in that case the timestamp will be updated on each call. Would it still be considered idempotent?

Reply:
Ah yes, `from` is optional. You're right, it would not be.